app-sidexis/DeleteDataContainer.ps1

```diff
@@ -0,0 +1,73 @@
+## PLEASE COMMENT YOUR VARIALBES DIRECTLY BELOW HERE IF YOU'RE RUNNING FROM A RMM
+## THIS IS HOW WE EASILY LET PEOPLE KNOW WHAT VARIABLES NEED SET IN THE RMM
+
+# Getting input from user if not running from RMM else set variables from RMM.
+
+
+$ScriptLogName = "DeleteDataContainer.log"
+
+if ($RMM -ne 1) {
+    $ValidInput = 0
+    # Checking for valid input.
+    while ($ValidInput -ne 1) {
+        # Ask for input here. This is the interactive area for getting variable information.
+        # Remember to make ValidInput = 1 whenever correct input is given.
+        $Description = Read-Host "Please enter the ticket # and, or your initials. Its used as the Description for the job"
+        if ($Description) {
+            $ValidInput = 1
+        } else {
+            Write-Host "Invalid input. Please try again."
+        }
+    }
+    $LogPath = "$ENV:WINDIR\logs\$ScriptLogName"
+
+} else { 
+    # Store the logs in the RMMScriptPath
+    if ($null -eq $RMMScriptPath) {
+        $LogPath = "$RMMScriptPath\logs\$ScriptLogName"
+        
+    } else {
+        $LogPath = "$ENV:WINDIR\logs\$ScriptLogName"
+        
+    }
+
+    if ($null -eq $Description) {
+        Write-Host "Description is null. This was most likely run automatically from the RMM and no information was passed."
+        $Description = "No Description"
+    }   
+
+
+    
+}
+
+# Start the script logic here. This is the part that actually gets done what you need done.
+
+# Path to the folder
+$FolderPath = "C:\datacontainer"
```

Comment:
Need to make this variable. Use $env:SYSTEMDRIVE for this instead of C:\

Reply:
If possible, query the registry or ini file for the application that references this.. since this can be configured within the application itself and it isn't static to the SYSTEMDRIVE.

Reply:
@dtcjoe please see comments before we merge this to main.

Reply:
requested changes